### PR TITLE
Add step installing cppo_ocamlbuild to fix xapi build

### PR DESCRIPTION
**Pull request by gaborigloi**
Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -50,6 +50,7 @@ Vagrant.configure("2") do |config|
     eval `opam config env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
     opam install -y lwt_react depext camlp4
```
Comment (mseri) [inline Vagrantfile]: Is this necessary?

Comment (gaborigloi) [reply Vagrantfile]: I can check if it builds without this line.

**Review by mseri — APPROVED** [review]
If that line is not needed please remove it and merge, otherwise feel free to merge

File: Vagrantfile (hunk 1 above)
Comment (gaborigloi) [reply Vagrantfile]: It seems that the build worked without that line.

**Pull request merged**